src/pages/workspaces/List.js

```diff
@@ -280,6 +280,7 @@ export const WorkspaceList = _.flow(
       const { workspace: { namespace, name } } = ws
       return Utils.textMatch(filter, `${namespace}/${name}`) && (includePublic || !ws.public || Utils.canWrite(ws.accessLevel))
```

Comment:
The message currently shows up if you have workspaces, but search for a non-existent string. We could move this `textMatch` down to the secondary filtering below to fix that (which would have a side effect of changing the list of available billing projects we show, but that's probably actually a beneficial change)